.circleci/config.yml

```diff
@@ -91,9 +91,6 @@ jobs:
       # Copy database config
       - run: cp config/database.yml.example config/database.yml
 
-      # Setup the environment
-      - run: cp .env.example .env
```

Comment:
I removed this because this file is not read from the Ruby docker image, so it was confusing.

Reply:
Ok!